package.json

```diff
@@ -36,31 +36,30 @@
     "@types/chai": "^4.2.0",
     "@types/fs-extra": "^5.1.0",
     "@types/mocha": "^9.1.0",
-    "@types/node": "^12.0.0",
+    "@types/node": "^18.11.9",
     "@typescript-eslint/eslint-plugin": "4.29.2",
     "@typescript-eslint/parser": "4.29.2",
     "chai": "^4.2.0",
     "eslint": "^7.29.0",
     "eslint-config-prettier": "8.3.0",
     "eslint-plugin-import": "2.24.1",
     "eslint-plugin-prettier": "3.4.0",
-    "ethereum-waffle": "^3.2.0",
+    "ethereum-waffle": "^4.0.7",
     "ethers": "^5.0.0",
+    "fs-extra": "^10.1.0",
     "hardhat": "^2.0.0",
     "mocha": "^10.0.0",
     "prettier": "2.4.1",
     "rimraf": "^3.0.2",
     "ts-node": "^8.1.0",
-    "typescript": "~4.5.2"
+    "typescript": "~4.5.2",
+    "@types/sinon-chai": "^3.2.3",
+    "@types/web3": "1.0.19"
   },
   "peerDependencies": {
     "@nomiclabs/hardhat-ethers": "^2.0.0",
-    "ethereum-waffle": "^3.2.0",
+    "ethereum-waffle": "*",
```

Comment:
```suggestion
    "ethereum-waffle": ">=4.0.7",
```
How about this?

Reply:
I think the plugin will also work with waffle 3.0, why putting additional constraints?

Reply:
right, makes sense.